build.gradle

```diff
@@ -43,7 +43,7 @@ plugins {
 
 allprojects {
     group 'bio.terra'
-    version '1.6.0-SNAPSHOT'
+    version '1.9.0-SNAPSHOT'
```

Comment:
Doesn't this get automatically bumped?

Reply:
Yes, but while I'm testing it bumps it on my branch.